src/java/org/apache/cassandra/index/sai/QueryContext.java

```diff
@@ -33,7 +33,7 @@
 @NotThreadSafe
 public class QueryContext
 {
-    private static final boolean DISABLE_TIMEOUT = Boolean.getBoolean("cassandra.sai.test.disable.timeout");
```

Comment:
is it for debugging?

Reply:
I hope so, that disables all query timeouts :)

Reply:
yeah, I added the disable to cassandra.yaml and nothing changed so I just ripped it out (that's why the commit is labeled FIXME)